### DISCUSSION TITLE
MeshCentral and MS Entra ID

### DISCUSSION THREAD

**Discussion by davidc-enc** [Ylianst/MeshCentral#5837] [Q&A]
Hello, could you please advise me? We use MS Office 365, and for user management, we use Entra ID. We are now testing MeshCentral, and it's incredible software, thank you so much for it! I need to set up permissions in MeshCentral according to groups from Entra ID. So, if a user who is in the "admin" group in MS Entra ID logs in, they should automatically be assigned to the admin group in MeshCentral. Additionally, members of one group in MS Entra ID should be able to see computers from another group. Is this possible, please? And if so, where and how can I set it up? Thanks

**Comment by mstrhakr** [comment]
This is supported with pull #5856
